src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/PublicationTests.cs

```diff
@@ -0,0 +1,160 @@
+                    },
+                    new()
+                    {
+                        Id = release2002Id,
+                        Version = 0,
+                        ReleaseName = "2002",
+                        Published = null,
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                    },
+                },
+            };
+
+            var result = publication.LatestPublishedRelease();
+            Assert.Equal(release2001Id, result.Id);
+        }
+
+        [Fact]
+        public void LatestRelease()
+        {
+            var release2000Id = Guid.NewGuid();
+            var release2000AmendmentId = Guid.NewGuid();
+            var release2001Id = Guid.NewGuid();
+            var release2001AmendmentId = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Releases = new List<Release>
+                {
+                    new()
+                    {
+                        Id = release2000Id,
+                        Version = 0,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-2),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+
+                    },
+                    new()
+                    {
+                        Id = release2000AmendmentId,
+                        Version = 1,
+                        PreviousVersionId = release2000Id,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-1),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    },
+                    new()
+                    {
+                        Id = release2001Id,
+                        Version = 0,
+                        ReleaseName = "2001",
+                        Published = DateTime.UtcNow.AddDays(-1),
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    },
+                    new()
+                    {
+                        Id = release2001AmendmentId,
+                        Version = 1,
+                        PreviousVersionId = release2001Id,
+                        ReleaseName = "2001",
+                        Published = null,
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                    },
+                },
+            };
+
+            var result = publication.LatestRelease();
+            Assert.Equal(release2001AmendmentId, result.Id);
+        }
+    }
```

Comment:
Like above I think you could add a `LatestRelease_NoRelease`. Has Publication.cs enabled nullable reference types yet? If so should be clearer to see that we can expect both these methods to return null.

Reply:
I've enabled nullable to Publication.cs and PublicationTests.cs. I had to make various changes to Publication.cs to make sure no changes appear if you make a migration - which I tested and a blank migration was created.